src/curl_xml.c

```diff
@@ -240,8 +240,9 @@ static int cx_check_type(const data_set_t *ds, cx_xpath_t *xpath) /* {{{ */
   }
 
   if (ds->ds_num != xpath->values_len) {
-    WARNING("curl_xml plugin: DataSet `%s' requires %zu values, but config "
-            "talks about %zu",
+    WARNING("curl_xml plugin: DataSet `%s' requires %" PRIsz
+            " values, but config "
+            "talks about %" PRIsz,
```

Comment:
Join with the previous line.

Reply:
Good call. clang-format formatted it this way, but it looks like it won't combine adjacent strings, only split long ones.